packages/dotcom-ui-header/src/__test__/components/__snapshots__/Drawer.spec.tsx.snap

```diff
@@ -130,9 +100,20 @@ exports[`dotcom-ui-header/src/components/Drawer renders ASK FT button 1`] = `
         className="o-header__drawer-menu-list"
       >
         <li
-          className="o-header__drawer-menu-item"
+          className="o-header__drawer-menu-item edition-switcher"
           data-trackable="edition-switcher"
         >
+          <span
+            className="current-edition"
+          >
+            UK
```

Comment:
Your change now puts edition name on the new line, while in previous snapshots `UK Edition` was on the same line

Reply:
no idea why because the snap shot is automatically generated and the html looks correct so... i dont think is relevant because in html that line dont have effect